apps/api/src/services/auth.service.ts

```diff
@@ -50,9 +53,20 @@ export class AuthService {
       session.nonce = undefined;
 
       // Attempt to find the user that matches the verified wallet address
-      const user = await findByWalletAddress(siweData.address);
-      const userId = user?.id;
       const wallet = siweData.address;
+      let user = await findByWalletAddress(wallet);
```

Comment:
the new auth logic where if you log in with a new wallet, we'll create the user if they dont already exist